src/github/client/github-client-interceptors.ts

```diff
@@ -138,6 +138,7 @@ export const handleFailedRequest = (logger: Logger) =>
 				logger.error(errorMessage);
 			}
 
+			logger.info({ response, request, error, isWarning }, "joshkay temp logging - githubclient interceptor handleFailedRequest");
```

Comment:
Want the actual error on Axios fail

Reply:
Is there a reason we use `info` instead of `error` for log level?

Reply:
purely just was re-using the same method for all of my logs. This one is dumping out everthing it gets its hands on so not just specifically errors. Plus the hidden lines above are throwing .errrors